From b7baefafe9d7459259b2ecb5a74b85f765a5e5a6 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sat, 19 Jul 2014 04:26:31 -0400
Subject: [agp] intel: Rename gtt_bus_addr to gtt_phys_addr

Message-id: <20140719042630.19438.38143.stgit@gir.stowe>
Patchwork-id: 85998
O-Subject: [RHEL7.1 PATCH 36/81] agp/intel: Rename gtt_bus_addr to gtt_phys_addr
Bugzilla: 1110895
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110895
Upstream Status: 5acc4ce44cd0a9cf5dbcfe50085708e9156e0177

commit 5acc4ce44cd0a9cf5dbcfe50085708e9156e0177
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Mon Jan 6 14:39:40 2014 -0700

    agp/intel: Rename gtt_bus_addr to gtt_phys_addr

    The only use of gtt_bus_addr is as an argument to ioremap(), so it is a CPU
    physical address, not a bus address.  Rename it to gtt_phys_addr to reflect
    this.

    No functional change.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Reviewed-by: Daniel Vetter <daniel.vetter@ffwll.ch>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/char/agp/intel-gtt.c b/drivers/char/agp/intel-gtt.c
index b8e2014..54202ff 100644
--- a/drivers/char/agp/intel-gtt.c
+++ b/drivers/char/agp/intel-gtt.c
@@ -64,7 +64,7 @@ static struct _intel_private {
  struct pci_dev *pcidev; /* device one */
  struct pci_dev *bridge_dev;
  u8 __iomem *registers;
- phys_addr_t gtt_bus_addr;
+ phys_addr_t gtt_phys_addr;
  u32 PGETBL_save;
  u32 __iomem *gtt;  /* I915G */
  bool clear_fake_agp; /* on first access via agp, fill with scratch */
@@ -191,7 +191,7 @@ static int i810_setup(void)
  writel(virt_to_phys(gtt_table) | I810_PGETBL_ENABLED,
         intel_private.registers+I810_PGETBL_CTL);
 
- intel_private.gtt_bus_addr = reg_addr + I810_PTE_BASE;
+ intel_private.gtt_phys_addr = reg_addr + I810_PTE_BASE;
 
  if ((readl(intel_private.registers+I810_DRAM_CTL)
   & I810_DRAM_ROW_0) == I810_DRAM_ROW_0_SDRAM) {
@@ -636,10 +636,10 @@ static int intel_gtt_init(void)
 
  intel_private.gtt = NULL;
  if (intel_gtt_can_wc())
-  intel_private.gtt = ioremap_wc(intel_private.gtt_bus_addr,
+  intel_private.gtt = ioremap_wc(intel_private.gtt_phys_addr,
             gtt_map_size);
  if (intel_private.gtt == NULL)
-  intel_private.gtt = ioremap(intel_private.gtt_bus_addr,
+  intel_private.gtt = ioremap(intel_private.gtt_phys_addr,
          gtt_map_size);
  if (intel_private.gtt == NULL) {
   intel_private.driver->cleanup();
@@ -796,7 +796,7 @@ static int i830_setup(void)
  if (!intel_private.registers)
   return -ENOMEM;
 
- intel_private.gtt_bus_addr = reg_addr + I810_PTE_BASE;
+ intel_private.gtt_phys_addr = reg_addr + I810_PTE_BASE;
 
  return 0;
 }
@@ -1123,13 +1123,13 @@ static int i9xx_setup(void)
  case 3:
   pci_read_config_dword(intel_private.pcidev,
           I915_PTEADDR, &gtt_addr);
-  intel_private.gtt_bus_addr = gtt_addr;
+  intel_private.gtt_phys_addr = gtt_addr;
   break;
  case 5:
-  intel_private.gtt_bus_addr = reg_addr + MB(2);
+  intel_private.gtt_phys_addr = reg_addr + MB(2);
   break;
  default:
-  intel_private.gtt_bus_addr = reg_addr + KB(512);
+  intel_private.gtt_phys_addr = reg_addr + KB(512);
   break;
  }
 
-- 
1.7.1